numpy/lib/npyio.py

```diff
@@ -967,8 +967,7 @@ def split_line(line):
         line = line.strip('\r\n')
         if line:
             return line.split(delimiter)
-        else:
-            return []
+        return []
```

Comment:
I'm not sold on this change or the one below being an improvement.

Reply:
Yeah, I guess it's simply a style change. How about 238a134? If IYO that's not an improvement either I'll remove those suggestions and just stick with the change to `_decode_line`.

Reply:
Slightly better, thanks - the `byte_converters` one is sort of net neutral to me too, but I don't feel too strongly about it. I suppose it's the difference between an `else` a "default", and I the distinction is fairly subjective.

Reply:
Agreed - I was aiming for conciseness but I don't think one way is necessarily better than the other. I'll keep this in mind when reading through things in the future as well :+1: